.github/workflows/reusable_track_size.yml

```diff
@@ -192,7 +192,7 @@ jobs:
       - name: Render benchmark result
         if: github.ref == 'refs/heads/main'
         run: |
-          python3 -m pip install google-cloud-storage==2.9.0
+          python3 -m pip install --break-system-package google-cloud-storage==2.9.0
```

Comment:
should be applied here https://github.com/rerun-io/rerun/blob/bd87d4c825922b007f0c8c9a176cc9617d40aa9f/.github/workflows/reusable_bench.yml#L161 as well